Exebite.API/Controllers/CommentsController.cs

```diff
@@ -42,6 +42,24 @@ public class CommentsController : ControllerBase
                    .Reduce(_ => UnprocessableEntity(), error => error is ValidationError)
                    .Reduce(_ => InternalServerError(), x => _logger.LogError(x.ToString()));
 
+        [HttpPost("reaction")]
+        [Authorize(Policy = nameof(AccessPolicy.CreateCommentReactionAccessPolicy))]
+        public IActionResult PostReaction([FromBody] InsertCommentReactionDto commentReaction) =>
+            _mapper.Map<CommentInsertReactionModel>(commentReaction)
+                   .Map(_commandRepository.InsertReaction)
+                   .Map(x => Created(new { saved = x }))
+                   .Reduce(_ => BadRequest(), error => error is ValidationError)
+                   .Reduce(_ => InternalServerError(), x => _logger.LogError(x.ToString()));
+
+        [HttpDelete("reaction")]
```

Comment:
It's common to provide the `{id}` of the record that should be deleted.

```suggestion
        [HttpDelete("reaction/{reactionId}")]
```